modules/complex-number/test/test_prokofeva_elizaveta_complex_number.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright 2021 Prokofeva Elizaveta
+
+#include <gtest/gtest.h>
+
+#include "include/complex_number.h"
+
+TEST(Prokofeva_Liza_Complex_Number, Test_Sum_And_Diff) {
+    double re1 = 1.5;
+    double im1 = 0.5;
+
+    double re2 = 2.5;
+    double im2 = 3.5;
+
+    double re3 = 0.5;
+    double im3 = -0.5;
+
+    ComplexNumber first(re1, im1);
+    ComplexNumber second(re2, im2);
+    ComplexNumber third(re3, im3);
+
+    ComplexNumber result = first + second - third;
+
+    ASSERT_DOUBLE_EQ(3.5, result.getRe());
+    ASSERT_DOUBLE_EQ(4.5, result.getIm());
+}
+
+TEST(Prokofeva_Liza_Complex_Number, Test_Mult_And_Diff) {
+    double re1 = 10.0;
+    double im1 = 2.9;
+
+    double re2 = 18.75;
+    double im2 = 23.46;
+
+    ComplexNumber first(re1, im1);
+    ComplexNumber second(re2, im2);
+
+    ComplexNumber result = (second * first) / (first * second);
+
+    ASSERT_EQ(1, result.getRe());
+    ASSERT_EQ(0, result.getIm());
+}
+
+TEST(Prokofeva_Liza_Complex_Number, Test_Equal_Mark) {
```

Comment:
В этом тесте ты, по большей части, проверяешь присваивание. Так что лучше сделать соответствующее название:
```suggestion
TEST(Prokofeva_Liza_Complex_Number, Test_Assignment) {
```